### ISSUE TITLE
Support network movement with frames

### ISSUE THREAD

**Issue by FallenRecruit** [CyclopsMC/IntegratedDynamics#443]
#### Issue type: Bug
- :bug: Bug
____
#### Actual behaviour: Crashes with error log

[crash-2017-12-08_18.55.49-server.txt](https://github.com/CyclopsMC/IntegratedDynamics/files/1541891/crash-2017-12-08_18.55.49-server.txt)

#### Steps to reproduce the problem:

![2017-12-08_19 49 31](https://user-images.githubusercontent.com/23091107/33758098-fbf31ca6-dc50-11e7-9468-fa5578a4a194.png)
1. Make funky locomotion frame contraption that pushes Energy exporter that is on a cable
2. Push frame

____
#### Versions:

- This mod: 0.9.6s
- Integrated Tunnels: 1.3.5
- CyclopsCore: 0.10.23 
- Minecraft: 1.12.2 
- Forge: 14.23.1.2555 
- Direwolf20: 1.2.0

#### Log file:

https://pastebin.com/5wzgu52Q

**Comment by rubensworks** [comment]
Yeah, this was to be expected, don't move ID networks with frames (for now). Most likely we'll have to add custom support for FL.

**Comment by josephcsible** [comment]
If you want to support this now, should we also reopen #309?

**Comment by rubensworks** [comment]
Quark doesn't provide an API for this AFAIK. So we'll have to wait and see if this can be solved without using the FL API, if so, this should automatically work with Quark as well.

**Comment by octylFractal** [comment]
I would appreciate if before if the network even worked while on a "frame mod", that it did not crash at all. It is sometimes hard to predict when certain blocks are going to be moved, and with the new "full block" status of cables, it's extremely easy to move cables by accident. Having this crash a server is confusing, and could be easily (I think) avoided by correctly removing parts from the network when the block entity is removed.

**Comment by rubensworks** [comment]
> Having this crash a server is confusing, and could be easily (I think) avoided by correctly removing parts from the network when the block entity is removed.

@octylFractal ID should already be cleaning up things correctly when blocks are removed. But you're welcome to submit a PR if you think anything needs to be changed 🙂

**Comment by octylFractal** [comment]
I would submit a PR, but the build seems to require me to have a username and password for a Maven repository (or maybe for me to install everything CyclopsMC from scratch), and your CLA requires write access to my gists. This is a non-starter for me, so I am unable to make a PR.

**Comment by rubensworks** [comment]
Should be working now thanks to the work around #1674.
